Replicator/Replicator.cc

```diff
@@ -846,8 +831,7 @@ namespace litecore { namespace repl {
                         auto error = C4Error::printf(WebSocketDomain, 404,
                                                      "Collection '%.*s' is not found on the remote server",
                                                      SPLAT(collPath));
-                        gotError(error);
-                        _stop();
+                        onError(error);
```

Comment:
Do we want to make it a permanent or transient error? With the PR, the error will be transient?

Reply:
OK, I think the replicator will stop() with 404 error.

Reply:
Yeah, _disconnect() is essentially the same as _stop(). _stop() just calls _disconnect